ow_power_system/include/power_system_node.h

```diff
@@ -5,20 +5,32 @@
 #include <chrono>
 
 #include "PrognoserFactory.h"
+#include <ow_faults/SystemFaults.h>
 
 class PowerSystemNode
 {
 public:
   void Run();
 
+  int LOW_VOLTAGE = 1;
+  int CAP_LOSS = 2;
+  int THERMAL_FAULT = 4;
```

Comment:
I think these would better be an enum rather than public class data, since they are effectively constants.  And member data are required to have an "m_" prefix.  If they are used only in the .cpp file, they can be defined there.

Reply:
updating to static constexpr ints instead.